wandb/sdk/internal/context.py

```diff
@@ -11,11 +11,11 @@
 
 class Context:
     _cancel_event: threading.Event
-    _debug_record: Optional[Record]
+    # _debug_record: Optional[Record]
```

Comment:
maybe add the comment about debug here as well?

Reply:
done